src/core/textures/ImageTexture.ts

```diff
@@ -134,6 +134,7 @@ export class ImageTexture extends Texture {
 
   async loadImageFallback(src: string, hasAlpha: boolean) {
     const img = new Image();
+    img.crossOrigin = 'anonymous';
```

Comment:
Before it was excluded for inline images:
```suggestion
        if (!(src.startsWith('data:'))) {
      img.crossOrigin = 'Anonymous';
    }
```

Reply:
yes! good catch.